smartsheet-csharp-sdk/main/Smartsheet/Api/SmartsheetBuilder.cs

```diff
@@ -93,7 +93,8 @@ public class SmartsheetBuilder
         /// 
         /// <para>It is a constant with Value "https://api.smartsheet.com/2.0".</para>
         /// </summary>
-        public const string DEFAULT_BASE_URI = "https://api.smartsheet.com/2.0/";
```

Comment:
These changes need to be reverted correct?

Reply:
whoops yeah, reverted. Is there a better way to set this via local environment variables?

Reply:
Not that I know of :(